### PR TITLE
Add initial code to form path on local level

### PR DESCRIPTION
**Pull request by dnesbitt61**
Update to use edgeinfo_offset for directed edges.
Update timing in pathtest to not do CPU clock usage but use std::chrono:high_resolution_clock so we get elapsed time (include I/O wait).

### CODE REVIEW

File: src/thor/formlocalpath.cc [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        }
+if (!(directededge->endnode() == prior_local_node)) {
+  LOG_ERROR("Prior end not found...");
+}
+
+        // Set the prior end node on the local level to startnode
+        prior_local_node = startnode;
+
+        // Add the directed edge graph Id to the path
+        edgesonpath.emplace_back(GraphId(startnode.tileid(), startnode.level(),
```
Comment (dgearhart) [inline src/thor/formlocalpath.cc]: can just use args

File: src/thor/pathtest/pathtest.cc [hunk 2]
```diff
@@ -190,17 +203,25 @@ int main(int argc, char *argv[]) {
...
+  t2 = std::chrono::high_resolution_clock::now();
+  msecs = std::chrono::duration_cast<std::chrono::milliseconds>(t2-t1).count();
+  LOG_INFO("PathTest took " + std::to_string(msecs) + " ms");
 
   // Try the the directions
+  t1 = std::chrono::high_resolution_clock::now();
   TripDirections trip_directions = DirectionsTest(trip_path, originloc, destloc);
+  t2 = std::chrono::high_resolution_clock::now();
+  msecs = std::chrono::duration_cast<std::chrono::milliseconds>(t2-t1).count();
+  LOG_INFO("TripDirections took " + std::to_string(msecs) + " ms");
```
Comment (dgearhart) [inline src/thor/pathtest/pathtest.cc]: thanks for adding

**Comment by dgearhart** [comment]
:+1:

File: src/thor/formlocalpath.cc (hunk 1 above)
Comment (dnesbitt61) [inline src/thor/formlocalpath.cc]: I'll get this on the next PR when I add the code to recover the shortcuts (just started that)

**Pull request merged**